src/main/java/io/github/thebusybiscuit/slimefun4/core/accessibility/ConcreteAccessLevel.java

```diff
@@ -0,0 +1,79 @@
+package io.github.thebusybiscuit.slimefun4.core.accessibility;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Concrete access levels are defined to be constant and always present.
+ * These levels should be used a fallback for when fine-grained access
+ * processing is not needed or unavailable.
+ *
+ * @author md5sha256
+ */
+public class ConcreteAccessLevel implements AccessLevel {
+
+    /**
+     * Represents an access level where no privileges are allocated.
+     */
+    public static final ConcreteAccessLevel NONE = new ConcreteAccessLevel("&cNone");
+
+    /**
+     * Represents an access level where all privileges are allocated.
+     */
+    public static final ConcreteAccessLevel FULL = new ConcreteAccessLevel("&bFull");
```

Comment:
This should be turned into an enum, you have a private constructor anyway so this should definitely be an enum here.

Reply:
The issue here lies in the fact that an enum has a `final compareTo` method which would clash. I originally intended for this to be an enum.

Reply:
Hmmm, I see.
Could always add your own compare method to the interface though

Reply:
I'll give that go.